src/DynamoRevit/DynamoRevit.cs

```diff
@@ -305,15 +305,44 @@ public Result ExecuteCommand(DynamoRevitCommandData commandData)
             //subscribe to the assembly load
             AppDomain.CurrentDomain.AssemblyLoad += AssemblyLoad;
 
+            // Launch main Dynamo directly when ShowUiKey is true.
+            if (CheckJournalForKey(commandData, JournalKeys.ShowUiKey, false))
+            {
+                try
+                {
+                    extCommandData = commandData;
+                    LoadDynamoWithoutSplashScreen();
+                }
+                catch (Exception ex)
```

Comment:
It seems the catch part is common between both condition, should we put the whole logic into try-catch so we only have one single try catch?

Reply:
Yes changed it, now we have a single try-catch block.